README-MULTISITE.md

```diff
@@ -505,28 +513,32 @@ The value of `endpoint` should be the endpoint of an RGW in the master zone of t
 Here is an example of a host_vars for a host containing 2 rgw_instances in a secondary zone:
 
 ```yaml
-rgw_zonemaster: false
-rgw_zonesecondary: true
-rgw_zonegroupmaster: true
-rgw_multisite_proto: "http"
 rgw_instances:
   - instance_name: rgw3
+    rgw_zonemaster: false
+    rgw_zonesecondary: true
+    rgw_zonegroupmaster: true
```

Comment:
@alimaredia I'm still very confused with the concept of `zonegroupmaster`.
In the CI, we have `rgw_zonegroumaster: false` when it's a secondary zone see https://github.com/ceph/ceph-ansible/blob/master/tests/functional/rgw-multisite/host_vars/osd0#L17-L19

Reply:
@guits the value of `rgw_zonegroupmaster` doesn't matter for secondary zones because when secondary zones are created the zonegroup they are in already exists, thus we don't need to specify whether or not that zonegroup is master. I think you can leave that as it is.

Reply:
so this parameter shouldn't be set at all for secondary zones it means

Reply:
It could be removed or it could be kept, there's really no way the value could be configured that is `wrong`. Maybe we should add that into the docs?